src/runtime/virtcontainers/pkg/compatoci/utils_test.go

```diff
@@ -97,6 +111,26 @@ func TestContainerCapabilities(t *testing.T) {
 	assert.Equal(t, c.Permitted, []string(nil))
 	assert.Equal(t, c.Inheritable, []string(nil))
 	assert.Equal(t, c.Ambient, []string(nil))
+
+	ociSpec.Process.Capabilities = interface{}("")
+
+	c, err = ContainerCapabilities(ociSpec)
+	assert.NotNil(t, err)
+	assert.Equal(t, c.Bounding, []string(nil))
```

Comment:
This check is fine, but a better way would be to check if len(c.Bounding) is 0. Similarly for other checks as well.

Reply:
Thanks, I will fix it